### ISSUE TITLE
reload after xtrigger arg change

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc-flow#6259]
Reportedly, `cylc vr` on a running workflow (i.e., reload) does not update the clock trigger time.  Stopping the scheduler first (i.e., restart) does get it right.  

[UPDATE]
Confirmed, using a minimal example:
```ini
[scheduling]
    initial cycle point = now
    [[xtriggers]]
       # x = wall_clock(offset=P0Y)
        x = wall_clock(offset=P1Y)
    [[graph]]
        PT1M = "@x => foo"
[runtime]
    [[foo]]
```

**Comment by hjoliver** [comment]
I think this probably affects all xtriggers, not just clock-triggers.

Draft fix posted.

**Comment by hjoliver** [comment]
Closed by #6263
